Mod1.Lection2.Hw2.Task2/Mod1.Lection2.Hw2.Task2/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿var arrA = new int[20];
+var arrB = new int[20];
+
+FillingArrays();
+Console.WriteLine($"Sorted array {string.Join(",", BubbleSortArr(arrB))}");
+
+void FillingArrays()
+{
+    var random = new Random();
+
+    for (var i = 0; i < arrA.Length; i++)
+    {
+        arrA[i] = random.Next(0, 1000);
+        Console.WriteLine($"Arr A, element {i}: {arrA[i]}");
+
+        var arrElement = arrA[i] <= 888 ? arrB[i] = arrA[i] : arrA[i] = random.Next(0, 888);
+        Console.WriteLine($"Arr B, element {i}: {arrElement}");
+    }
+}
+
+static void Swap(ref int leftNum, ref int rightNum)
+{
+    (leftNum, rightNum) = (rightNum, leftNum); /*var temp = leftNum; leftNum = rightNum; rightNum = temp;*/
+}
+
+static int[] BubbleSortArr(int[] arrB)
+{
+    for (var i = 0; i < arrB.Length; i++)
+    {
+        for (var j = 0; j < arrB.Length - 1; j++)
+        {
+            if (arrB[j] < arrB[j + 1])
+            {
+                Swap(ref arrB[j], ref arrB[j + 1]);
+            }
+        }
+    }
+    return arrB;
```

Comment:
Since you swap references (mutating existing array) you do not really need to return anything.